### ISSUE TITLE
v1.22.2 of Kubernetes breaks litmus event tracker pod/server

### ISSUE THREAD

**Issue by tmonautoguy** [litmuschaos/litmus#3241]
**What happened**:
Upon installing litmus on a minikube cluster it was noticed that the event-tracker pod (litmuschaos/litmusportal-event-tracker:2.1.0) failed to start and eventually went into crashloopbackoff status. Additionally, it was not possible to queue workflows or watch them in the Chaos Center. Eventually the subscriber pod(litmuschaos/litmusportal-subscriber:2.1.0) will fail too.
**What you expected to happen**:
For the install to work correctly and for everything to work. :) 

**How to reproduce it (as minimally and precisely as possible)**:
Start minikube with version V1.22.2 of kubernetes i.e. `minikube start --kubernetes-version v1.22.2`, follow the entire getting started [doc](https://litmuschaos.github.io/tutorials/tutorial-getting-started/index.html#0). After logging into the Chaos Center, event tracker is deployed and will eventually fail.
**Anything else we need to know?**:
Logs from the failed event tracker pod:
```
k logs event-tracker-5b56b9fbcc-txfvr -n chaos
time="2021-09-02T07:33:49Z" level=info msg="Go Version: go1.16.7"
time="2021-09-02T07:33:49Z" level=info msg="Go OS/Arch: linux/amd64"
2021-09-02T07:33:49.674Z	INFO	controller-runtime.metrics	metrics server is starting to listen	{"addr": ":8080"}
2021-09-02T07:33:49.674Z	INFO	controller-runtime.injectors-warning	Injectors are deprecated, and will be removed in v0.10.x
2021-09-02T07:33:49.674Z	INFO	controller-runtime.injectors-warning	Injectors are deprecated, and will be removed in v0.10.x
2021-09-02T07:33:49.674Z	INFO	setup	starting manager
2021-09-02T07:33:49.674Z	INFO	controller-runtime.manager	starting metrics server	{"path": "/metrics"}
2021-09-02T07:33:49.675Z	INFO	controller-runtime.manager.controller.eventtrackerpolicy	Starting EventSource	{"reconciler group": "eventtracker.litmuschaos.io", "reconciler kind": "EventTrackerPolicy", "source": "kind source: /, Kind="}
I0902 07:33:50.818866       1 request.go:655] Throttling request took 1.046922188s, request: GET:https://10.0.0.1:443/apis/policy/v1beta1?timeout=32s
2021-09-02T07:33:51.671Z	ERROR	controller-runtime.source	if kind is a CRD, it should be installed before calling Start	{"kind": "EventTrackerPolicy.eventtracker.litmuschaos.io", "error": "no matches for kind \"EventTrackerPolicy\" in version \"eventtracker.litmuschaos.io/v1\""}
github.com/go-logr/zapr.(*zapLogger).Error
	/go/pkg/mod/github.com/go-logr/zapr@v0.2.0/zapr.go:132
sigs.k8s.io/controller-runtime/pkg/source.(*Kind).Start
	/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.8.0/pkg/source/source.go:117
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func1
	/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.8.0/pkg/internal/controller/controller.go:168
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start
	/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.8.0/pkg/internal/controller/controller.go:218
sigs.k8s.io/controller-runtime/pkg/manager.(*controllerManager).startRunnable.func1
	/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.8.0/pkg/manager/internal.go:659
2021-09-02T07:33:51.671Z	ERROR	setup	problem running manager	{"error": "no matches for kind \"EventTrackerPolicy\" in version \"eventtracker.litmuschaos.io/v1\""}
github.com/go-logr/zapr.(*zapLogger).Error
	/go/pkg/mod/github.com/go-logr/zapr@v0.2.0/zapr.go:132
main.main
	/event-tracker/main.go:139
runtime.main
	/usr/local/go/src/runtime/proc.go:225
```

**Comment by ksatchit** [comment]
cc: @rajdas98 any comment?

**Comment by songrgg** [comment]
I met the same issue and found the root cause should be "The apiextensions.k8s.io/v1beta1 API version of CustomResourceDefinition is no longer served as of v1.22.", so the installation of the CRDs failed. 

As [K8s v1beta1 deprecation](https://kubernetes.io/docs/reference/using-api/deprecation-guide/#customresourcedefinition-v122) says, we should migrate manifests and API clients to use the apiextensions.k8s.io/v1 API version, available since v1.16.

**Comment by songrgg** [comment]
For your case, to make the litmus run in minikube, you can downgrade the cluster to a version under 1.22, I used 1.18 and worked fine for me.

**Comment by tmonautoguy** [comment]
Thanks @songrgg! I am currently running v1.21.3 and that worked. Will we be able to fix this issue since eventually v1.21.3 will be deprecated?

**Comment by tomtrapp** [comment]
+1
That really blocks a lot

**Comment by rajdas98** [comment]
Hi all, It got fixed in Litmus 2.4.0. Please upgrade this component with 2.4.0
